.github/workflows/push_master.workflow.yml

```
@@ -142,8 +170,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: "12.x"
```

To be up to date with kuzzle-runner
```suggestion
          node-version: "14.x"
```